WebRoot/css/zm.css

```diff
@@ -5242,9 +5242,24 @@ If you feel compelled to have a background color, please do so on a per skin lev
 }
 
 .ZmAttachDialog-container .ZmBriefcaseItemSmall {
-	border-style:solid; 
-	border-width:1px 0px;
-	border-color:@lighten(AppC,60)@
+	@AttachBriefcaseItem@
+}
+
+.ZmAttachDialog-container .ZmBriefcaseItemSmall.Row-selected {
+	@AttachBriefcaseItemSelected@
+}
+
+.ZmAttachDialog-container .ZmBriefcaseItemSmall td:nth-child(2),
+.ZmAttachDialog-container .ZmBriefcaseItemSmall td:nth-child(3) {
+	@AttachBriefcaseItemChild@
+}
+
+.ZmAttachDialog-container .ZmBriefcaseItemSmall td:first-child {
```

Comment:
I feel there is no risk to use `:first-child` and `:last-child` pseudo selectors for such things, unless we target for major style declarations.